postgres/schema.sql

```diff
@@ -1,4 +1,5 @@
 CREATE TABLE IF NOT EXISTS responses (tx_id uuid PRIMARY KEY, ts timestamp WITH time zone DEFAULT NOW(), invalid boolean DEFAULT NULL, data jsonb);
+CREATE TABLE IF NOT EXISTS feedback_responses (id SERIAL, ts timestamp WITH time zone DEFAULT NOW(), invalid boolean DEFAULT NULL, data jsonb, survey char(25), period char(25));
```

Comment:
I'm not sure if you can create a column with data type char without having a size and 25 chars seemed sensible. I can change the SQLAlchemy to restrict the size to 25? Unless there's a better way to do this?